find.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import os.path
+
+parser = argparse.ArgumentParser(description='find all relevant files to include in a hackpack')
```

Comment:
I'll submit a pull request when I get when I get a chance.  Don't let the comments to this file block the release.